recipes/cs_libguarded/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
```

Comment:
cmake targets requires CMake 3+. I would prefer keeping cmake 2.8 and using ${CONAN_LIBS} instead.